ployst/ui/views.py

```diff
@@ -14,6 +15,7 @@ def get_context_data(self, *args, **kwargs):
         providers = get_all_providers_meta()
         data.update({
             'providers': providers,
+            'core_url': settings.CORE_API_ADDRESS
```

Comment:
I added this for debugging purposes, but it could either be deleted, generalised, or left for `DEBUG=True`, or replaced with debug toolbar.